docs/guides/admin/docs/configuration/https/opencast.only.md

```diff
@@ -1,24 +1,5 @@
-HTTPS
-=====
-
-This document will assist you in enabling HTTPS for a new or an existing
-Opencast installation, either via HTTPS termination proxy or directly in
-Opencast.
-
-Changing the domain name of an Opencast installation is a similar procedure.
-
-
-Motivation: HTTPS for Opencast, why?
-------------------------------------
-
-Your site visitor's privacy. While ISPs and public HotSpot operators and those
-who sniff unencrypted WiFi traffic might be able to see your visitors
-connecting to your site, they can't see, which video is watched when HTTPS is
-enabled.
-
-
-Enable HTTPS directly in Opencast – without proxy
--------------------------------------------------
+Enable HTTPS directly in Opencast
```

Comment:
The guide for enabling HTTPS inside of Opencast now got its own separate documentation page.